.simple-git-hooks.cjs

```diff
@@ -1 +0,0 @@
-module.exports = require('@1stg/simple-git-hooks')
```

Comment:
No hooks anymore?

Reply:
Correct.

Reply:
I don't think so, it prevents unexpected codes formatting and incorrect commit message for example.

Reply:
- `npm test` does that too
- hooks can be overwritten
- hooks don’t work on PRs?
- people will make mistakes, whether hooks are enabled or not

I can imagine they are useful when building applications or large projects.
This is a tiny package that should not require a lot of maintainance.

Reply:
> npm test does that too

It still needs to be called manually?

> hooks can be overwritten
> hooks don’t work on PRs?

We install hooks on `prepare` hook, so if the user install dependencies, it will never be overwritten and work on PRs.

> people will make mistakes, whether hooks are enabled or not

That's why we have CI workflow.

> This is a tiny package that should not require a lot of maintainance.

I'd like to be consistent for any scale of projects.

Reply:
All unified projects are already consistent. I don’t believe hooks will make them more consistent.

If you feel strongly about this, I am open to having it in this project.

I’d be against adding it to every unified project. I don’t see the value in it.

Reply:
> I’d be against adding it to every unified project. I don’t see the value in it.

Thanks, I'm not asking to using them in all unified projects.